File: C#/TestPalindrome/MorePalindromeTest.cs
using NUnit.Framework;
using us.fatehi.palindrome;

namespace us.fatehi.palindrome.test
{

    [TestFixture]
    public class MorePalindromeTest
    {
        private readonly Palindrome _p = new Palindrome();

        [Test]
        public void AlmostPalindrome()
        {
            Assert.IsFalse(_p.IsPalindrome("polyp"));
        }
    }
}

File: Java/src/test/java/us/fatehi/palindrome/MorePalindromeTest.java
package us.fatehi.palindrome;

import static org.junit.jupiter.api.Assertions.assertFalse;

import org.junit.jupiter.api.Test;

public class MorePalindromeTest {

  private final Palindrome p = new Palindrome();

  @Test
  public void almostPalindrome() {
    assertFalse(p.isPalindrome("polyp"));
  }

}
